uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Page/UserMenu.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+use CRM_HRCore_CMSData_CMSPathsFactory as CMSPathsFactory;
```

Comment:
sorry I think I wasn't clear, but what I meant here https://github.com/civicrm/civihr/pull/2227#discussion_r145736349 

is just to rename the alias to contain CMS so "CMSPathsFactory" instead of "PathsFactory" as you did here but not changing the actual class name from "CRM_HRCore_CMSData_PathsFactory" to "CRM_HRCore_CMSData_CMSPathsFactory " since CMS is repeated twice here.